client/lib/plans/features-list.js

```diff
@@ -835,7 +835,7 @@ export const FEATURES_LIST = {
 	[ constants.FEATURE_VIDEO_HOSTING_JETPACK ]: {
 		getSlug: () => constants.FEATURE_VIDEO_HOSTING_JETPACK,
 		getTitle: () => i18n.translate( 'Video Hosting' ),
-		getDescription: () => i18n.translate( '13Gb of high-speed, HD, and ad-free video hosting.' ),
+		getDescription: () => i18n.translate( '13 GB of high-speed, HD, and ad-free video hosting.' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.